var/spack/repos/builtin/packages/ports-of-call/package.py

```diff
@@ -10,13 +10,13 @@ class PortsOfCall(CMakePackage, CudaPackage):
     """Ports of Call: Performance Portability Utilities"""
 
     homepage    = "https://github.com/lanl/ports-of-call"
-    url         = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.0.0.tar.gz"
+    url         = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.1.0.tar.gz"
```

Comment:
This change can be reverted, since the `url` is only used as a template for finding new versions.